core/src/main/scala/org/http4s/syntax/AllSyntax.scala

```diff
@@ -4,8 +4,6 @@ package syntax
 trait AllSyntax
```

Comment:
Shouldn't `Message.messSyntax` be included here?

Reply:
:+1:  I don't want to add to the import tax.